examples/TableValuedParameters.js

```diff
@@ -0,0 +1,151 @@
+const Connection = require('../lib/tedious').Connection;
+const Request = require('../lib/tedious').Request;
+const TYPES = require('tedious').TYPES;
+
+const config = {
+  server: '192.168.1.212',
+  authentication: {
+    type: 'default',
+    options: {
+      userName: 'test',
+      password: 'test'
+    }
+  },
+  options: {
+    port: 1433 // Default Port
+  }
+};
+
+const connection = new Connection(config);
+
+const storedProcedure = '[dbo].[test_sp_tvp]';
+const table = '[dbo].[test_tvp]';
+const table_type = 'TableType';
+
+
+connection.on('connect', (err) => {
+  if (err) {
+    console.log('connection err');
+    throw err;
+  } else {
+    createTable();
+  }
+});
+
+// Creating new table called [dbo].[test_proced]
+//--------------------------------------------------------------------------------
+function createTable() {
+  const sql =
+    `CREATE TABLE ${table} (
+        user_id INT,
+        user_name VARCHAR(50),
+        user_enabled BIT
+        )`;
+
+  const request = new Request(sql, (err, rowCount) => {
+    if (err) {
+      throw err;
+    } else {
+      console.log(`'${table}' created!`);
+    }
+  });
+
+  request.on('requestCompleted', () => {
```

Comment:
I think it'd be better not to advertise the `requestCompleted` callback too much, but instead use the callback passed to `Request` instead. 🤔

Reply:
I moved these 'requesCompleted' section into the Request call backs :)